galsim/deprecated/shear.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2012-2014 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+import galsim
+from galsim.deprecated import depr
+
+def Shear_setE1E2(self, e1=0.0, e2=0.0):
+    """Deprecated method that will be deprecated eventually."""
```

Comment:
Deprecated method that will be deprecated eventually?  That sounds kind of weird and redundant to me.  How about "Deprecated method to set the Shear to some (e1, e2)"?  (Same comment applies to the other things in shear.py)